src/activity_handlers.cpp

```diff
@@ -499,7 +499,9 @@ int butcher_time_to_cut( const player &u, const item &corpse_item, const butcher
     if( corpse_item.has_flag( "QUARTERED" ) ) {
         time_to_cut /= 4;
     }
-
+    const auto helpers = g->u.get_crafting_helpers();
+    const int helpersize = std::max( 3, static_cast<int>( helpers.size() ) );
```

Comment:
```suggestion
    const int helpersize = std::min( 3, static_cast<int>( helpers.size() ) );
```
You want the smaller of the two values, right?  because if helpers.size == 0, your version returns 3.